app/src/main/java/javakotlinpractical/Interafce_abstractclass.kt

```diff
@@ -0,0 +1,41 @@
+package javakotlinpractical
+
+fun main(){
+    var button = Button()
+    button.touch()
+    button.click()
+    button.method1()
+}
+
+abstract  class abstractclass{
+    abstract fun method1()
+}
+interface MyFirstInterface{
+    fun touch()
+    fun click(){
+        println("click : MyFirstInterface")
+    }
+}
+interface MySecondInterface{
+    fun touch(){
+        println("touch : MySecondInterface")
+    }
+    fun click(){
+        println("click : MySecondInterface")
+    }
+}
+class Button : MyFirstInterface,MySecondInterface,abstractclass(){
```

Comment:
class Button : MyFirstInterface,MySecondInterface,abstractclass(){
this should be like :
class Button : MyFirstInterface, MySecondInterface, abstractclass() {